### ISSUE TITLE
About Multi-Label Change Detection

### ISSUE THREAD

**Issue by COUJIALUO** [dyzy41/ChangeCLIP#13]
If my dataset is for multi-label change detection, and the labels contain values that are no longer just (0 1) or (0 255), but rather like (0 1 2 3 4 5 6) for the labels, does your code still support multi-class change detection?

**Comment by dyzy41** [comment]
Yes, you should modify the num_classes in the config files. And, notice this line. 
https://github.com/dyzy41/ChangeCLIP/blob/4b9bb9a4241d6bd473c154752ddfdbdbb943fd93/mmseg/datasets/transforms/loading.py#L125

**Comment by qilong164765913** [comment]
你好，如果是想用SECOND这样的多标签数据集训练需要对label进行合并吗？因为他们给了time1和time2两个阶段的label，但我看您的数据集都只有一个label标记变换区域，能麻烦您解答下我该怎么处理这种两个时期的label，使其能在您的项目上训练

**Comment by qilong164765913** [comment]
> 你好，如果是想用SECOND这样的多标签数据集训练需要对label进行合并吗？因为他们给了time1和time2两个阶段的label，但我看你的数据集都只有一个label标记变换区域，能麻烦您解答下我该怎么处理这个两个时期的label，使其能在你的项目上训练

@dyzy41
